aiida/orm/authinfo.py

```diff
@@ -11,20 +11,23 @@
 
 from aiida.transport import TransportFactory
 from aiida.common.exceptions import (ConfigurationError, MissingPluginError)
+from .backend import Collection, CollectionEntry
 
 __all__ = ['AuthInfo', 'AuthInfoCollection']
 
 
-class AuthInfoCollection(object):
+class AuthInfoCollection(Collection):
+    """The collection of AuthInfo entries."""
+
     __metaclass__ = abc.ABCMeta
 
     @abc.abstractmethod
     def create(self, computer, user):
         """
         Create a AuthInfo given a computer and a user
 
-        :param computer: A Computer or DbComputer instance
-        :param user: A User or DbUser instance
+        :param computer: a Computer or DbComputer instance
```

Comment:
Sorry, I thought we were going to leave out allowing a DbX to be passed into these?

Reply:
Forgot to adapt the docstring here. Fixed